modules/apps/portlet-configuration/portlet-configuration-web/src/main/resources/META-INF/resources/js/PermissionsCheckbox.js

```diff
@@ -35,18 +35,18 @@ export default function PermissionsCheckbox({
 			onChange={() => {
 				setChecked((prevCheckedState) => !prevCheckedState);
 
-				const changeCount = Liferay.Util.SessionStorage.getItem(
-					`${_portletNamespace}changeCount`,
+				const checkedPermissionChangeCount = Liferay.Util.SessionStorage.getItem(
```

Comment:
should be permissionCheckboxChangeCount?

Reply:
Thank you!